RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -408,8 +408,14 @@ boolean auto_pre_adventure()
 				adjustForYellowRayIfPossible(mon);
 				zoneHasWantedMonsters = true;
 			}
+			if(auto_wantToBanish(monster_phylum(mon), place))
+			{
+				// attempt to prepare for banishing, but if we can not try free running
+				boolean canBanish = adjustForBanishIfPossible(monster_phylum(mon), place);
```

Comment:
`canBanish` does not seem to be used anywhere after this. Does it need to be assigned?